packages/docusaurus-plugin-content-pages/src/index.ts

```diff
@@ -132,6 +179,18 @@ export default function pluginContentPages(
           title: frontMatter.title ?? contentTitle,
           description: frontMatter.description ?? excerpt,
           frontMatter,
+          lastUpdatedBy: lastUpdatedData.lastUpdatedBy,
+          lastUpdatedAt: lastUpdatedData.lastUpdatedAt,
+          editUrl: getItemEditUrl({
+            editUrl,
+            itemDirPath: pagesDirPath,
+            itemSourceAbsolute: pagesSourceAbsolute,
+            permalink,
+            i18n,
+            options,
+            siteDir,
+            contentPaths,
+          }),
```

Comment:
😄 maybe that was a bad idea, if we can only use it on the pages plugin and not all other plugins it's probably not worth it to extract to docusaurus utils 😅  (didn't see at first that the edit URL fn all have different param names)